metering/configuring-metering/metering-configure-aws-billing-correlation.adoc

```diff
@@ -28,19 +28,19 @@ spec:
     spec:
       config:
         aws:
-          secretName: "your-aws-secret"
+          secretName: "your-aws-secret" <2>
```

Comment:
@lamek Can you review the updates to this file? The build was giving warnings that there were callouts below these code examples, but no callout in the actual code for these.

Reply:
@bergerhoffer, @lamek, the variables in both these files should be enclosed in angle brackets.

Reply:
@kalexand-rh This PR was just to fix the build warnings, but I can add the brackets.

Reply:
LGTM.